autovalet_system/launch/state_machine.launch

```diff
@@ -30,7 +30,7 @@
 
         <!-- STATE MACHINE -->
         <!-- state machine node runs lane detection, goal generation, and parking subsystems, as well as overall system logic -->
-        <node name="STATE_MACHINE" pkg="autovalet_system" type="run_sm.sh" args="$(arg simulation)" />
+        <!-- <node name="STATE_MACHINE" pkg="autovalet_system" type="run_sm.sh" args="$(arg simulation)" /> -->
```

Comment:
don't change this? we still want a not-debugging version on `master`.

Reply:
yes i had it uncommented, just haven't pushed that change yet